public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,30 @@
+using SplashKitSDK;
+
+namespace draw_line_on_window

```

Comment:
For the namespace, I suggest using the function name in Pascal case, followed by "Example" or something like that (as snake case is not used in C#).